ui/packages/components/src/hooks/useDebounce.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useMemo, useRef } from 'react';
+//@ts-ignore
+import debounce from 'lodash.debounce';
+
+const useDebounce = (callback: () => void, delay: number = 1000) => {
```

Comment:
This is copied from the the dev server - what's the plan for these @anafilipadealmeida - should I refactor to drop the old one and use the shared hook?